charts/trento-server/charts/trento-web/templates/ingress.yaml

```diff
@@ -22,6 +22,9 @@ metadata:
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- if .Values.global.certManager.enabled }}
+    cert-manager.io/cluster-issuer: "{{ .Values.global.certManager.clusterIssuer }}"
+  {{- end }}
```

Comment:
this as well should be moved to the values override file (again, without the if block).